### PR TITLE
User can settle synths explicitly by UI

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
On L1, when one trades 12 consecutive trades into a specific synth, it is required to settle this synth in order to make the 13th attempt. The system will automatically block any further attempts before the settlement. We add a "settle" button and a user prompt (also a link to the doc of fee reclamation) to allow the user to settle synths by themselves on the Kwenta UI.

## Related issue
#321 

## Motivation and Context

Before this change, the user can't settle synths by themselves on the Kwenta UI.

## How Has This Been Tested?

1. User swaps from sUSD to sJPY 12 times
2. The UI shows a message "You have 12 consecutive trades into sJPY. Please trade out of sJPY or settle:” and a button "Settle". (There is a waiting period after completing the trade. After that, user can attempt to settle) 
4. Click on the button "Settle"
5. A pop-up showing "Settle sJPY" and a transaction is submitted to the user's wallet.
6. User confirms the transaction.
7. After a few minutes, the user can swap from sUSD into sJPY again.

## Screenshots (if appropriate):
message
image
metamask

### CODE REVIEW

**Comment by LeifuChen** [comment]
I have tested it twice, and the code is working. However,  I have the following comments about the implementation.
1. The current UI is unable to refresh automatically after the settlement. Instead, it requires manual refresh (e.g., click the arrows in the middle). So I need some help to make it better.
2. Do we want to show the "submitting order" after clicking the "settle" button? The button text won't change right now. 

I hope I can have some input from your side. Thanks.

File: sections/exchange/hooks/useExchange.tsx [hunk 1]
```diff
@@ -718,6 +727,46 @@ const useExchange = ({
...
+				const tx = await synthetixjs.contracts.Exchanger.settle(
+					walletAddress,
+					ethers.utils.formatBytes32String(baseCurrencyKey as string),
+					gas
+				);
+
+				if (tx != null) {
+					monitorTransaction({
+						txHash: tx.hash,
+						onTxConfirmed: () => {
```
Comment (JChiaramonte7) [inline sections/exchange/hooks/useExchange.tsx:754]: > The current UI is unable to refresh automatically after the settlement. Instead, it requires manual refresh (e.g., click the arrows in the middle). So I need some help to make it better.
Do we want to show the "submitting order" after clicking the "settle" button? The button text won't change right now.

In `onTxConfirmed` you could also try to invalidate the `useNumEntriesQuery` query. This should hopefully return 0 now that all transactions have been settled.

https://react-query.tanstack.com/guides/query-invalidation

**Review by JChiaramonte7 — COMMENTED** [review]
Tested and works great!

File: sections/exchange/hooks/useExchange.tsx (hunk 1 above)
Comment (LeifuChen) [reply sections/exchange/hooks/useExchange.tsx:756]: @JChiaramonte7 Thanks for your input. It reminds me to use `refetch()` to update the state before and after the settlement. The new change has been pushed.

**Comment by LeifuChen** [comment]
The countdown timer of the settlement waiting period has been added.
image

**Pull request merged**